### ISSUE TITLE
Error message when getting connector table unspecific

### ISSUE THREAD

**Issue by eichlerkathi** [natverse/rcatmaid#156]
When using catmaid_get_connector_table() for a neuron that is missing pre- or postsynaptic sites and not specifying direction, the error message reads "Error in catmaid_fetch(path, body = post_data, conn = conn, ...) : Bad Request (HTTP 400).". 
This could be more specific saying that neuron is lacking pre- or post-connectors and to specify the direction with incoming or outgoing.
@jefferis

**Comment by jefferis** [comment]
Thanks @eichlerkathi. Could you give an example skid that still fails?

**Comment by eichlerkathi** [comment]
Yes, sorry forgot to include. Skid 10944609 has no presynaptic sites and fails.
